platform-db/src/main/java/com/softicar/platform/db/runtime/table/logic/save/DbTableRowSaverSlave.java

```diff
@@ -51,6 +53,17 @@ public void afterSave() {
 			sendNotification(IDbTableListener::afterSave, rows);
 			DbTableRowCommitNotifier.addNotification(table, DbTableRowNotificationType.SAVE, rows);
 		}
+		if (!changedRows.isEmpty()) {
+			DbTableRowCommitNotifier.addNotification(table, DbTableRowNotificationType.CHANGE, changedRows);
+		}
+	}
+
+	private Collection<R> getRowsWithPersistentChange(Collection<R> rows) {
```

Comment:
This mehtod name got me confused. How about plain and simple `getChangedRows`, or `determineChangedRows`?

Reply:
- it is not just `ChangedRows` because it does not only look for `row.dataChanged()` but also `row.impermanent()`
- open for suggestions

Reply:
We kind of overloaded the semantics of "change" here: On the one hand we have `DbTableRowFlag.DATA_CHANGED`, and on the other hand we have `DbTableRowNotificationType.CHANGE`.

We could resolve this in a brutally explicit way:
`this.rowsForChangeNotification = determineRowsForChangeNotification(rows);`

Not quite elegant but at least it leaves no room for interpretation.

Reply:
good idea, changed as requested